src/sentry_freight/plugin.py

```diff
@@ -53,6 +53,12 @@ class FreightPlugin(ReleaseTrackingPlugin):
     description = 'Integrate Freight release tracking.'
     version = sentry_freight.VERSION
 
+    def has_project_conf(self):
```

Comment:
is this going to cause the plugin to have that `Configure plugin` link on the `All integrations` page?